hackernews/program.py

```diff
@@ -33,20 +35,18 @@ def on_slack_command(event):
         fetch_articles(topic, all_articles)
         new_articles = all_articles - current_articles
 
-        if new_articles:
-            for article in new_articles:
-                _, title, url = article
-                s_message = f"Title: {title}, URL: {url if url else 'No URL'}"
-                print(s_message)
-                slack.chat_postMessage(channel=SLACK_CHANNEL, text=s_message)
-            current_articles.update(new_articles)
+        for article in new_articles:
+            _, title, url = article
+            slack_message = f"Title: {title}, URL: {url if url else 'No URL'}"
+            slack.chat_postMessage(channel=event.data.channel, text=slack_message)
+        current_articles.update(new_articles)
 
         time.sleep(120)
```

Comment:
make the time configurable

Reply:
@mario99logic: i.e. add a const at the top, maybe name it `POLLING_INTERVAL_SECS`, or maybe even add it as a project var and parse into an int in the Python const.